src/atmi-backward-compat.cmake

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) 2016 - 2021 Advanced Micro Devices, Inc. All Rights Reserved.
```

Comment:
since its a new file, just 2022 is enough here

Reply:
Done